PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -487,9 +516,24 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     docString = cms.string("MC matching to status==1 electrons or photons"),
 )
 
+electronMCTableNew = cms.EDProducer("CandMCMatchTableProducer",
+    src     = electronTable.src,
+    mcMapDressedLep = cms.InputTag("electronsMCMatchForTableAlt"),
+    mcMap   = cms.InputTag("electronsMCMatchForTable"),
+    mapTauAnc = cms.InputTag("matchingElecPhoton:hasTauAnc"),
+    objName = electronTable.name,
+    objType = cms.string("ElectronDressed"), 
+    branchName = cms.string("genPart"),
+    docString = cms.string("MC matching to status==1 electrons or photons"),
+    genparticles     = cms.InputTag("finalGenParticles"), 
+)
+
+
 electronSequence = cms.Sequence(bitmapVIDForEle + bitmapVIDForEleHEEP + isoForEle + ptRatioRelForEle + seedGainEle + slimmedElectronsWithUserData + finalElectrons)
 electronTables = cms.Sequence (electronMVATTH + electronTable)
 electronMC = cms.Sequence(electronsMCMatchForTable + electronMCTable)
+electronMCnew = cms.Sequence(particleLevelForMatching + tautaggerForMatching + matchingElecPhoton + electronsMCMatchForTable + electronsMCMatchForTableAlt + electronMCTableNew)
+( run2_nanoAOD_106Xv1 & ~run2_nanoAOD_devel).toReplaceWith(electronMC, electronMCnew)
```

Comment:
we should have the new sequence by default and then for the v8 ( run2_nanoAOD_106Xv1 & ~run2_nanoAOD_devel) we should run the old sequence.

can you confirm that runs on both
runTheMatrix -l 1325.81 ( #test ttbar 106Xv1 ) 
and runTheMatrix -l 1325.7 ( # test ttbar 94XMiniAODv2)

Reply:
thanks for the clarification, i misunderstood the meaning of the modifiers. I've just pushed the proper setting, along with some fixes. I confirm the runTheMatrix workflows are running properly